app/models/event_stream.rb

```diff
@@ -130,7 +78,6 @@ def group_level
     @group_and_level ||= self.class.group_and_level(event_type)
   end
 
-  # TODO: Consider moving since this is EmsEvent specific. group, group_level and group_name exposed as a virtual columns for reports/api.
   def group_name
     @group_name ||= self.class.group_name(group)
```

Comment:
Should there be an abstract version of group_name at the class level in the EventStream base class?

Reply:
Added, see 👇